quickstart/troubleshooting.md

```diff
@@ -295,3 +295,18 @@ Thread 1 (Thread 0x7f782559f040 (LWP 56)):
 {% endtab %}
 {% endtabs %}
 
+-------------
+
+##  Diskpool behaviour  
+
+The below behaviour may be encountered while uprading from older releases to Mayastor 2.4 release and above. Use the following commands for resolution: 
+
+### Get Dsp
+
+Running `kubectl get dsp -n mayastor` could result in the error due to the `v1alpha1` schema in the discovery cache. To resolve this, run the command `kubectl get diskpools.openebs.io -n mayastor`. After this kubectl discovery cache will be updated with `v1beta1` object for dsp. 
+ 
+### Create API
+
+When creating a Disk Pool with `kubectl create -f dsp.yaml`, you might encounter an error related to `v1alpha1` CR definitions. To resolve this, ensure your CR definition is updated to `v1beta1` in the YAML file (for example, `apiVersion: openebs.io/v1beta1`).
+
+User can validate above listed schema changes by executing ```kubectl get crd diskpools.openebs.io``` post migration.
```

Comment:
Let's remove "post migration"

Reply:
post upgrade to 2.4 or later @avishnu ?

Reply:
Since it's already mentioned in line 302, I was thinking, we can remove the phrase "post migration" as it's implicit.